extension/src/websocket-forwarder.ts

```diff
@@ -35,16 +35,26 @@ export class WebSocketForwarder implements vscode.Disposable {
     });
   }
 
-  private handleClientMessage(message: string) {
-    const obj = JSON.parse(message);
```

Comment:
ah not even needed?

Reply:
nop, if no stringify is run under client writer